labs/lab1/chapter1.md

```diff
@@ -18,85 +18,80 @@ Expected completion: 15-20 minutes
 
 Agenda:
 
-* Review Docker and systemd
-* Review Docker help
+* Review docker and systemd
+* Review docker help
 * Explore a Dockerfile
 * Build an image
 * Launch a container
 * Inspect a container
-* Build Docker registry
+* Build docker registry
 
 
-## Docker and systemd
+Perform the following commands as student unless instructed otherwise.
+
+## docker and systemd
 
-Check out the systemd unit file that starts Docker on the CDK and notice that 
-it includes 3 EnvironmentFiles. These files tell Docker how the Docker daemon, 
+Check out the systemd unit file that starts docker on Minishift and notice that 
+it includes 3 EnvironmentFiles. These files tell docker how the docker daemon, 
 storage and networking should be set up and configured. Take a look at those 
 files too. Specifically, in the /etc/sysconfig/docker file check out the registry 
 settings. You may find it interesting that you can `ADD_REGISTRY` and 
 `BLOCK_REGISTRY`. Think about the different use cases for that.
 
-Perform the following commands as student unless instructed otherwise.
-
-XXX we want to consider the new model of using docker from minishift
-in this section. Please run the following command to get access to the
-mihnishift docker
-
 ```bash
-eval $(minishift docker-env)
+minishift ssh "cat /usr/lib/systemd/system/docker.service"
+minishift ssh "cat /usr/lib/systemd/system/docker-storage-setup.service"
+minishift ssh "cat /etc/sysconfig/docker"
+minishift ssh "cat /etc/sysconfig/docker-storage"
+minishift ssh "cat /etc/sysconfig/docker-network"
 ```
 
+Now check the status of docker within minishift.
 
 ```bash
-cat /usr/lib/systemd/system/docker.service
-cat /usr/lib/systemd/system/docker-storage-setup.service
-cat /etc/sysconfig/docker
-cat /etc/sysconfig/docker-storage
-cat /etc/sysconfig/docker-network
+minishift ssh "systemctl status docker"
```

Comment:
wait... nm, i see what you mean now... 
looks like its not really referencing those configs... setting everything in service file here:
```shell
$ minishift ssh "cat /etc/systemd/system/docker.service"
[Unit]
Description=Docker Application Container Engine
Documentation=http://docs.docker.com
After=network.target rc-local.service
Requires=rhel-push-plugin.socket
Requires=docker-cleanup.timer

[Service]
Type=notify
NotifyAccess=all
Environment=GOTRACEBACK=crash
Environment=DOCKER_HTTP_HOST_COMPAT=1
Environment=PATH=/usr/libexec/docker:/usr/bin:/usr/sbin
ExecStart=/usr/bin/dockerd-current -H tcp://0.0.0.0:2376 -H unix:///var/run/docker.sock            --authorization-plugin rhel-push-plugin            --selinux-enabled            --log-driver=journald            --add-runtime docker-runc=/usr/libexec/docker/docker-runc-current            --default-runtime=docker-runc            --exec-opt native.cgroupdriver=systemd            --userland-proxy-path=/usr/libexec/docker/docker-proxy-current            --add-registry registry.access.redhat.com            --storage-driver devicemapper --tlsverify --tlscacert /etc/docker/ca.pem            --tlscert /etc/docker/server.pem --tlskey /etc/docker/server-key.pem            --label provider=kvm --insecure-registry 172.30.0.0/16 --insecure-registry 192.168.0.0/16 
ExecReload=/bin/kill -s HUP 
LimitNOFILE=1048576
LimitNPROC=1048576
LimitCORE=infinity
TimeoutStartSec=0
Delegate=yes
KillMode=process
MountFlags=slave
Environment=

[Install]
WantedBy=multi-user.target
```